packages/lib-classifier/src/components/Classifier/components/ImageToolbar/ImageToolbar.js

```diff
@@ -12,39 +11,37 @@ import ZoomInButton from './components/ZoomInButton'
 import ZoomOutButton from './components/ZoomOutButton'
 import withKeyZoom from '../withKeyZoom'
 
-class ImageToolbar extends Component {
-  render () {
-    return (
-      <Box height='min-content' {...this.props}>
```

Comment:
This allows you to pass the standard Grommet styling props to the toolbar. So different layouts can use different padding etc.

Why was it removed? Easy enough to restore, though.

Reply:
It was removed because as Shaun mentioned, I couldn’t find cases where props were explicitly passed to ImageToolbar either. However, when investigating sticky behavior, it turns that passing  generalized `…props` in the ImageToolbar component file is how the sticky css rules are applied to the toolbar `div` via DefaultLayout. I added a code comment in ImageToolbar as a reference to how the css `…props` are passed from the page layout.